go.sum

```diff
@@ -56,6 +59,7 @@ github.com/shoenig/atomicfs v0.1.1 h1:MYIOQjG1qIW5J1dsI1dginEVGC9lnDTOtls9sKNHm4
 github.com/shoenig/atomicfs v0.1.1/go.mod h1:aw1MEIbywlavaTMRvnai/OLCh2dV5XWNcNBYf6iLDno=
 github.com/shoenig/httplus v0.0.0 h1:WrkAZNjv4Sv6juOB4Mb0KhTTlyKIPkODiCaAU3pnMCI=
 github.com/shoenig/httplus v0.0.0/go.mod h1:1RATT1GFHcr+r7R/Bjc5Z266PHtyukMD6avWsREh0Is=
+github.com/shoenig/petrify/v4 v4.1.0/go.mod h1:b89FOhyhZQN3Av042hXzf/Bj49MsR7kZzO3tgiYs2NM=
```

Comment:
an old version of petrify, lets figure out why this got added and fix it

Reply:
once removed, it doesn't come back. :man_shrugging: